src/main/java/com/rocketden/main/Main.java

```diff
@@ -1,9 +1,12 @@
 package com.rocketden.main;
 
+import com.ulisesbocchio.jasyptspringboot.annotation.EnableEncryptableProperties;
+
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-@SpringBootApplication
+@SpringBootApplication()
```

Comment:
Out of curiosity, why the parentheses? Not an issue.

Reply:
Sorry, this was leftover from other changes I was trying to make -- removed 👍